src/services/amendCourtCase/applyAmendmentsToAho.test.ts

```diff
@@ -92,17 +92,19 @@ describe("applyAmendmentsToAho", () => {
   it("applies OffenceReasonSequence amendments to aho", () => {
     const offenceIndex = 3
     const amendments = {
-      offenceReasonSequence: {
-        offenceIndex,
-        updatedValue: "newOffenceReasonSequenceValue"
-      }
+      offenceReasonSequence: [
+        {
+          offenceIndex,
+          updatedValue: "newOffenceReasonSequenceValue"
+        }
+      ]
     } as Amendments
 
     aho.AnnotatedHearingOutcome.HearingOutcome.Case.HearingDefendant.Offence = [
-      dummyOffence,
-      dummyOffence,
-      dummyOffence,
-      dummyOffence
+      cloneDeep(dummyOffence),
```

Comment:
We have to use `cloneDeep()` to avoid problems with deeply nested objects values being passed by reference